api/proto/metrics/metrics.proto

```diff
@@ -173,6 +173,7 @@ message appCpuMetric {
   uint64 total_ns = 7;                 // like 'total' but in nanoseconds. Overflow may happen after ~40 years without EVE restart
 }
 
+// StorageStatus - DEPRECATED. DO NOT USE THIS. Moved to info
```

Comment:
Has any code been written in zedcloud or eden? It was added just a week ago, right? So we might be able to just delete the added messages.

Reply:
We did not include it into Eden and I cannot see any entries on zedcloud side.

Reply:
As for the delete,  there are problems with yetus, and here, in order to avoid them, is it possible to leave it as it is for now? Or is there another way to solve these problems?
> Has any code been written in zedcloud or eden? It was added just a week ago, right? So we might be able to just delete the added messages.

Reply:
We can ignore those yetus complaints if we are convinced that nothing has started using the new API. (We judiciously ignore yetus complaints - after careful review etc).